hail/python/hail/docs/Makefile

```diff
@@ -50,7 +50,7 @@ clean:
 .PHONY: html
 html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
-	rsync -avz tutorials/iframe_figures/ $(BUILDDIR)/html/tutorials/iframe_figures/ # https://github.com/plotly/plotly.py/blob/master/packages/python/plotly/plotly/io/_base_renderers.py#L522-L529
+	[ ! -e tutorials/iframe_figures/ ] || rsync -avz tutorials/iframe_figures/ $(BUILDDIR)/html/tutorials/iframe_figures/ # https://github.com/plotly/plotly.py/blob/master/packages/python/plotly/plotly/io/_base_renderers.py#L522-L529
```

Comment:
`make -C hail-docs-no-test` doesn't generate any iframe_figures so the rsync fails (because the directory doesn't exist).